logservice.go

```diff
@@ -42,8 +42,27 @@ func parseFlags() app {
 	flag.StringVar(&a.buildID, "build", "", "ID of the build that is emitting logs ($SD_BUILDID)")
 	flag.StringVar(&a.token, "token", "", "JWT for authenticating with the Store API ($SD_TOKEN)")
 	flag.IntVar(&a.linesPerFile, "lines-per-file", defaultLinesPerFile, "Max number of lines per file when uploading ($SD_LINESPERFILE)")
+	flag.BoolVar(&a.isLocal, "local-mode", false, "Build run in local mode")
+	flag.StringVar(&a.buildLogFile, "build-log-file", "", "Path to the build log file in local mode")
 	flag.Parse()
 
+	if len(os.Getenv("SD_LINESPERFILE")) != 0 {
+		l, err := strconv.Atoi(os.Getenv("SD_LINESPERFILE"))
+		a.linesPerFile = l
```

Comment:
This assignment should be after the error check?

Reply:
I fixed it.